jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialectIT.java

```diff
@@ -278,6 +278,27 @@ void fullOuterJoin() throws SQLException {
         assertFalse(result.next());
     }
 
+    @Test
+    void rightJoinWithComplexCondition() throws SQLException {
+        final String query = String
+                .format("SELECT * FROM  %1$s.t1 a RIGHT OUTER JOIN  %1$s.t2 b ON a.x||a.y=b.x||b.y ORDER BY a.x", VIRTUAL_SCHEMA);
+        final ResultSet result = executeQuery(query);
+        matchNextRow(result, (long) 2, "bbb", (long) 2, "bbb");
```

Comment:
Wrap consecutive asserts (even hidden ones) in `assertAll()` to improve error localization and quality of test output.

Reply:
done